content/en/docs/contributing/docs/style-guide.md

```diff
@@ -0,0 +1,400 @@
+
+The semantic purpose of a data table is to present tabular data. Sighted users can quickly scan the table but a screen reader goes through line by line. A table caption is used to create a descriptive title for a data table. Assistive technologies (AT) use the HTML table caption element to identify the table contents to the user within the page structure.
+
+- Add table captions using [Hugo shortcodes](/docs/contribute/style/hugo-shortcodes/#table-captions) for tables.
+
+## Content best practices
+
+### Use present tense
+
+Do | Don't
+:--| :-----
+This command starts a proxy. | This command will start a proxy.
+
+### Use active voice
+
+Exception: Use passive voice if active voice leads to an awkward construction.
+
+Do | Don't
+:--| :-----
+You can explore the API using a browser. | The API can be explored using a browser.
+The YAML file specifies the replica count. | The replica count is specified in the YAML file.
+
+### Use simple and direct language
+
+Use simple and direct language. Avoid using unnecessary phrases, such as saying "please."
+
+Do | Don't
+:--| :-----
+To create a ReplicaSet, ... | In order to create a ReplicaSet, ...
+See the configuration file. | Please see the configuration file.
+View the pods. | With this next command, we'll view the pods.
+
+### Address the reader as "you"
+
+Do | Don't
+:--| :-----
+You can create a Kustomization by ... | We'll create a Kustomization by ...
+In the preceding output, you can see... | In the preceding output, we can see ...
+
+### Avoid Latin phrases
+
+Prefer English terms over Latin abbreviations.
+
+| Do          | Don't |
+|-------------|-------|
+| That is     | i.e   |
+| For example | e.g.  |
+
+Exception: Use "etc." for et cetera.
+
+## Patterns to avoid
+
+### Avoid using we
+
+Using "we" in a sentence can be confusing, because the reader might not know whether they're part of the "we" you're describing.
+
+Do | Don't
+:--| :-----
+Version 2 includes ... | In version 2, we have added ...
+Flux provides a new feature for ... | We provide a new feature ...
+This page teaches you how to use Kustomizations. | In this page, we are going to learn about Kustomizations.
+
+### Avoid jargon and idioms
+
+Some readers speak English as a second language. Avoid jargon and idioms to help them understand better.
+
+Do | Don't
+:--| :-----
+Internally, ... | Under the hood, ...
+Create a new cluster. | Turn up a new cluster.
+
+### Avoid statements about the future
+
+Avoid making promises or giving hints about the future. If you need to talk about an alpha feature, put the text under a heading that identifies it as alpha information.
+
+An exception to this rule is documentation about announced deprecations targeting removal in future versions.
+
+### Avoid statements that will soon be out of date
+
+Avoid words like "currently" and "new." A feature that is new today might not be considered new in a few months.
+
+Do | Don't
+:--| :-----
+In version 2, ... | In the current version, ...
```

Comment:
In some places we were trying to dial a back a little on the use of v1 and v2 - not sure how to reconcile with this guideline.  Generally I agree that it makes sense though.

Reply:
When referring to flux v1, the only way to reconcile with this guide is to say

In Flux v1, rather than, in the old version of Flux

and clarify in the documentation, where we use this style of specifying the version.

We should only have to use this versioning in the migration section